modules/atomic-widgets/widgets/atomic-image.php

```diff
@@ -29,13 +30,19 @@ public function get_name() {
 	protected function render() {
 		$settings = $this->get_atomic_settings();
 
-		$image_url = $settings['image'];
+		if ( empty( $settings['image'] ) ) {
+			return;
```

Comment:
@yotamselementor not sure it's related to this PR but as i understood we will not support having empty url right? should always have some placeholder

Reply:
Good point, fixed: https://github.com/elementor/elementor/pull/28626/commits/ce528555002965302bd918842c725c9e258ef692